.env.test

```diff
@@ -29,6 +30,7 @@ GOOGLE_CSE_KEY=xxx_google_cse_key_xxx
 GRAVITY_ID=xxx_artsy_id_xxx
 GRAVITY_SECRET=xxx_artsy_secret_xxx
 IMPULSE_APPLICATION_ID=xxx_impulse_id_xxx
+EXCHANGE_APP_ID=xxx_stress_id_xxx
```

Comment:
#superminor since I know this isn't functional but but `xxx_exchange_id_xxx`?

Reply:
ooh good catch! not sure why our replace all `stress` with `exchange` didn't catch this!